src/pages/components/pagination/accessibility.mdx

```diff
@@ -103,18 +100,20 @@ selection. When the previous and next page arrow buttons have focus, both
 </Row>
 
 If an overflow ellipsis is present in the pagination nav, once focused, pressing
-Space activates the menu to choose an available page to navigate to. Enter or
-Tab selects a value from the menu and closes it. The menu can also be closed by
-pressing Esc.
+`Space` or `Up` or `Down` arrows activates the select menu to choose an
+available page to navigate to. `Up` or `Down` arrows navigate between page
+options in the menu. `Space` or `Enter` selects a page from the menu and closes
+it. The menu can also be closed by pressing `Esc`.
 
 <Row>
 <Column colLg={12}>
 
-![The menu opens by pressing Space and pressing Tab or Enter selects a page from the menu and closes it.](images/pagination-a11y-4.png)
+![The menu opens by pressing Space or Up or Down arrows and pressing Space or Enter selects a page from
+  the select menu and closes it.](images/pagination-a11y-4.png)
 
 <Caption>
-  The menu opens by pressing Space and pressing Tab or Enter selects a page from
-  the menu and closes it.
+  The menu opens by pressing Space or Up or Down arrows and pressing Space or
```

Comment:
Maybe "The menu opens by pressing Space, Up arrow, or Down arrow, and closes by pressing Space or Enter to select an item."